api/clients/client.go

```diff
@@ -0,0 +1,71 @@
+//
+// Copyright (c) 2020 SSH Communications Security Inc.
+//
+// All rights reserved.
+//
+
+package clients
+
+import (
+	"github.com/SSHcom/privx-sdk-go/restapi"
+)
+
+// Client is a role-store client instance.
+type Client struct {
+	api restapi.Connector
+}
+
+// New creates a new user-store client instance
+func New(api restapi.Connector) *Client {
+	return &Client{api: api}
+}
+
+// CreateTrustedClient registers new client to PrivX
+func (clients *Client) CreateTrustedClient(client TrustedClient) (string, error) {
```

Comment:
Five times client in the same line. Maybe the `clients` should be called as `userstore`

Reply:
Should we rename it everywhere? Somehow we started to use type `Client` for all REST API clients. May be yes...